motion-of-no-confidence.md: Put teeth into the committee responsibilities

wking opened the pull request:
I don't expect us to need these teeth, but there's not much point in
having a list of responsibilities without (potential) consequences for
violating them.  This gives both the steering committee and the
Foundation membership at large the ability to force early elections if
they feel that the a member or members of the current committee are
neglecting their responsibilities.

I shortened the usual 90-day lead time for the early elections,
because that seemed to be too large a fraction of the annual
term. Between 45 and 60 days should provide both a sufficiently long
period for campaigning and a sufficiently short period for minimizing
further negligence.

Fixes #9.

wking commented:
The “Motion of no-confidence” wording is from [Wikipedia][1], which also outlines a few alternative approaches (e.g. Germany [proposes a successor in the same motion][2]).

[1]: http://en.wikipedia.org/wiki/Motion_of_no_confidence
[2]: http://en.wikipedia.org/wiki/Motion_of_no_confidence#Germany

katyhuff commented on motion-of-no-confidence.md:
```diff
@@ -0,0 +1,32 @@
+unlikely event that a committee member doesn't [attend 75% of the
+year's meetings][minimum-attendance], the other committee members and
+the Software Carpentry community at large should be able to evict the
+bum.
```
I am not a fan of the word bum.

wking replied:
On Thu, Mar 12, 2015 at 12:50:52PM -0700, Katy Huff wrote:
> > +the Software Carpentry community at large should be able to evict the
> > +bum.
> 
> I am not a fan of the word bum.

Maybe “delinquent member”?

katyhuff commented on motion-of-no-confidence.md:
```diff
@@ -0,0 +1,32 @@
+
+The steering committee has [responsibilities][], and it's important
+that those responsibilities are enforceable.  For example, in the
+unlikely event that a committee member doesn't [attend 75% of the
```
Necessary to say unlikely? Let's stick to the facts in this document as much as possible.

wking replied:
On Thu, Mar 12, 2015 at 12:51:20PM -0700, Katy Huff wrote:
> > @@ -0,0 +1,32 @@
> > +# Motion of no confidence
> > +
> > +The steering committee has [responsibilities][], and it's important
> > +that those responsibilities are enforceable.  For example, in the
> > +unlikely event that a committee member doesn't [attend 75% of the
> 
> Necessary to say unlikely? Let's stick to the facts in this document
> as much as possible.

Dropped in d056dc0 (motion-of-no-confidence.md: Drop 'unlikely',
2015-03-12).

katyhuff commented on motion-of-no-confidence.md:
```diff
@@ -0,0 +1,32 @@
+If at any time the committee passes (following their usual voting
+rules) a motion of no confidence or the [Foundation
+membership][membership] submits a petition signed by over half of the
+members, early elections must be immediately scheduled.  The early
```
Suggestion of more concise rephrasing:

"An early election must be immediately scheduled if (1)the steering committee passes a motion of no confidence or (2) the Foundation Membership submits a petition signed by over half of the Foundation Members.:

wking replied:
On Thu, Mar 12, 2015 at 12:54:44PM -0700, Katy Huff wrote:
> > +If at any time the committee passes (following their usual voting
> > +rules) a motion of no confidence or the [Foundation
> > +membership][membership] submits a petition signed by over half of the
> > +members, early elections must be immediately scheduled.  The early
>
> Suggestion of more concise rephrasing:
>
> "An early election must be immediately scheduled if (1)the steering
> committee passes a motion of no confidence or (2) the Foundation
> Membership submits a petition signed by over half of the Foundation
> Members."

Adopted with some capitalization tweaks in ebae756
(motion-of-no-confidence.md: Itemize early election triggers,
2015-03-12).  I've explained my feelings on capitalization in the
commit message, but I'm happy to adopt whatever conventions the board
feels are appropriate.

katyhuff commented:
Just a couple of general suggestions to add to the specific line comments:

- More brevity in this rule would be good if possible.
- All instances of "committee" or "steering committee" should be "Steering Committee"

katyhuff commented on motion-of-no-confidence.md:
```diff
@@ -0,0 +1,33 @@
+that those responsibilities are enforceable.  For example, in the
+event that a committee member doesn't [attend 75% of the year's
+meetings][minimum-attendance], the other committee members and the
+Software Carpentry community at large should be able to evict the bum.
```
delinquent member works.

wking commented:
On Thu, Mar 12, 2015 at 12:57:12PM -0700, Katy Huff wrote:
> - More brevity in this rule would be good if possible.

I'll try ;).  We can drop the whole intro paragraph if you like, but I
think the example provides helpful context.  I'd also welcome advice
on simplifying the rule, if you have anything specific you'd like to
change.

> - All instances of "committee" or "steering committee" should be
>   "Steering Committee"

Ah, that sounds like a consistent capitalization convention ;).  I'll
rebase and force-push.

wking commented:
On Thu, Mar 12, 2015 at 12:57:12PM -0700, Katy Huff wrote:
> - All instances of "committee" or "steering committee" should be
>   "Steering Committee"

Ok, I've rebased the capitalization correction and the “delinquent
member” bit into the first commit of the series, because the former is
something only lawyers will care about, and the second is something
that is maybe insensitive.  Neither deserves a place in the history (I
don't think).  I've kept the subsequent changes (“Drop 'unlikely'” and
“Itemize early election triggers”) because I felt like I had useful
things to say in their commit messages, and I wanted the history to
record that there had been discussion and iterative improvements to
the wording.  Let me know if you'd prefer a different reroll.  Folks
who want to view the changes since the initial review can try things
like:

  $ git diff --word-diff=color 06492bc..e387c35

jiffyclub commented:
I'm fine to have the SC vote on this.

katyhuff commented:
@wking - can you resolve the conflicts with master?

wking commented:
On Thu, Mar 12, 2015 at 04:14:12PM -0700, Katy Huff wrote:
> can you resolve the conflicts with master?

Rebased.  I added a governance.md#rules-of-order anchor to catch up
with #22.

katyhuff commented:
thx!

katyhuff commented:
Approved by SCF SC.

katyhuff commented:
Thanks @wking !

wking commented:
On Thu, Mar 26, 2015 at 01:23:43PM -0700, Katy Huff wrote:
> Thanks @wking !

Happy to help :).
